src/features/generic_events/aggregate/user-actions/aggregated-user-action.js

```diff
@@ -0,0 +1,33 @@
+import { RAGE_CLICK_THRESHOLD_EVENTS, RAGE_CLICK_THRESHOLD_MS } from '../../constants'
+
+export class AggregatedUserAction {
+  constructor (evt, selectorPath) {
+    this.event = evt
+    this.count = 1
+    this.originMs = Math.floor(evt.timeStamp)
+    this.relativeMs = [0]
```

Comment:
We could just not send this if count = 1. Trims this off payload size?

Reply:
since those relative values are potentially critical for a UI experience, i hesitate to remove them for some cases and not others